.eslintrc.json

```diff
@@ -54,6 +60,8 @@
         "react/jsx-filename-extension": "error",
         "react/prefer-stateless-function": "warn",
         "react/sort-comp": "warn",
-        "react/jsx-max-props-per-line": [1, { "when": "multiline" }]
+        "react/jsx-max-props-per-line": [1, { "when": "multiline" }],
+        "import/namespace": "off",
+        "import/default": "off"
```

Comment:
Another problem that seems to occur with two config files and using gulp-eslint to run the linter is as follows. So basically I was using two config files and 2 tasks to run the linter for the test file and the normal src files. The problem arises when I run the linter for the src files, it also lints the test files as the test files are of the form *.js. I know a workaround but it will require the the use of [gulp-ignore](https://www.npmjs.com/package/gulp-ignore). Or we could use the 2nd approach suggested by @Sanjo .